src/test/controller/PromotionController.test.ts

```diff
@@ -153,12 +154,13 @@ describe('Unit tests for PromotionController', function () {
     const expectedPromotion = new PromotionFactory().generateWithRelatedEntities(
       user
     );
+    const inputtedPromotion = setGooglePlacesAddress(expectedPromotion);
```

Comment:
Whats an inputtedPromotion? Maybe promotionRequestBody

Reply:
I put input as in that's the input promotion we're sending in, in contrast to expected/output/result